frontend/app/package.json

```diff
@@ -4,45 +4,46 @@
   "description": "A portfolio tracking, asset analytics and tax reporting application specializing in Cryptoassets that protects your privacy",
   "author": "Rotki Solutions GmbH <info@rotki.com>",
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "test:unit": "vue-cli-service test:unit",
-    "test:e2e": "vue-cli-service test:e2e --mode e2e",
-    "lint": "vue-cli-service lint --ext .js,.ts,.vue,.json",
-    "electron:build": "vue-cli-service electron:build",
-    "electron:serve": "vue-cli-service electron:serve",
-    "i18n:report": "vue-cli-service i18n:report --src './src/**/*.?(ts|js|vue)' --locales './src/locales/**/*.json'",
-    "lint:check": "vue-cli-service lint --no-fix --max-warnings 0  --ext .js,.ts,.vue,.json",
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run --headless",
+    "serve": "node scripts/serve.js --web",
+    "build": "cross-env NODE_ENV=production node scripts/build.js",
+    "test:unit": "cross-env TZ=UTC vitest --dir tests/unit",
+    "test:e2e": "dotenv -e .env.e2e start-server-and-test serve http-get://localhost:8080 cypress:open",
+    "lint": "eslint ./ --ext .js,.ts,.vue,.json",
+    "electron:build": "npm run build && electron-builder build --config .electron-builder.config.js",
+    "electron:serve": "node scripts/serve.js",
+    "i18n:report": "vue-i18n-extract report --vueFiles './src/**/*.?(ts|js|vue)' --languageFiles './src/locales/**/*.json'",
+    "lint:check": "eslint ./ --no-fix --max-warnings 0  --ext .js,.ts,.vue,.json",
     "preinstall": "node ../check-versions.js",
     "postinstall": "electron-builder install-app-deps",
     "postuninstall": "electron-builder install-app-deps",
     "serve:backend": "node start-backend.js",
     "lint:style": "stylelint 'src/**/*.{vue,scss}'",
     "lint:style:fix": "stylelint --fix 'src/**/*.{vue,scss}'",
-    "test:e2e-ci": "nyc vue-cli-service test:e2e --mode e2e --headless",
+    "test:e2e-ci": "dotenv -e .env.e2e start-server-and-test serve http-get://localhost:8080 cypress:run",
```

Comment:
Do you think we can get the coverage working with cypress now that we got rid of webpack?

Reply:
I created an issue for this #4391